python/foglamp/services/core/api/configuration.py

```diff
@@ -84,6 +84,10 @@
     :Example:
             curl -d '{"key": "TEST", "description": "description", "value": {"info": {"description": "Test", "type": "boolean", "default": "true"}}}' -X POST http://localhost:8081/foglamp/category
     """
+    keep_original_items = None
+    if 'keep_original_items' in request.query and urllib.parse.unquote(request.query['keep_original_items']) != '':
```

Comment:
@amarendra-dianomic why did we need this `urllib.parse.unquote` for `request.query['keep_original_items']`

Reply:
It is a query parameter and, principally, it should always be parsed and validated before using its value. We should not assume that user will always supply the correct and valid value.

Reply:
we  allow `true` or `false`  ONLY HENCE NO! We really don't care as those are not valid requests. and should not be respected and check is not required. 
aiohttp router will send us with proper encoding as is to us.